src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -4,6 +4,7 @@ module Nri.Ui.AssignmentIcon.V1 exposing
     , selfReview
     , peerReview, submitting, rating, revising
     , startPrimary, startSecondary
+    , allLibrary, assessment, assignArrow, home, passageQuiz, planningDiagnostic, practice2, quiz2, standards, unitDiagnostic, writing
```

Comment:
💭 it seems a bit questionable to me to have `allLibrary`, `home`, and `assignArrow` be grouped as "assignment icons".  I wonder if there's a better place to put those.  Do we know where else they might get used?

💭 `practice2`, `quiz2`: I wonder if there are better names that would better describe the difference in purpose of these vs the existing icons.  Do we plan to phase out the old practice and quiz icons soon?

Reply:
agreed!  I was struggling with naming.  🤦🏼‍♀️